src/Value.php

```diff
@@ -60,10 +60,8 @@ public function __construct($value, Container\Movable $container)
 
     /**
      * Countable
-     *
-     * @return int
      */
-    public function count()
+    public function count(): int
```

Comment:
Problem (bc break): adding return types to classes that are not `final` can break child class implementations.

Let's use `#[\ReturnTypeWillChange]` instead of declaring `:bool`?